librz/core/canalysis.c

```diff
@@ -6851,3 +6851,61 @@ RZ_API RZ_OWN RzPVector *rz_core_analysis_bytes(RZ_NONNULL RzCore *core, RZ_NONN
 	}
 	return vec;
 }
+
+/**
+ * \brief Set analysis hint offset
```

Comment:
Set analysis hint for the first immediate of the instruction at current offset to \p struct_member.